src/sentry/api/endpoints/debug_files.py

```diff
@@ -311,7 +319,7 @@ def delete(self, request: Request, project) -> Response:
         :auth: required
         """
 
-        if request.GET.get("id") and (request.access.has_scope("project:write")):
```

Comment:
Originally, I didn't understand why they had this here but it's because the endpoint uses `ProjectReleasePermission` which has:
```
"DELETE": ["project:admin", "project:releases"],
```
Regular members have `project:releases` letting anyone access this endpoint, so they needed an additional check to ensure only Admins/Managers/Owners could delete files. I've updated this check to allow team admins to also do this now